doc/data/messages/i/invalid-sequence-index/bad.py

```diff
@@ -0,0 +1,4 @@
+fruits = ['apple', 'banana', 'orange']
+print(fruits['apple'])  # [invalid-sequence-index]
+# but be careful with data[True], it will work since True is represnted as 1
+# and False as 0, so it will work
```

Comment:
You can create a details.rst to add this information is you want. If you think pylint should raise in this case (arguably yes, then it's a false negative that we could fix).

Reply:
Okey, so let's move them to `details.rst`. Thank you very much.